usbms.c

```diff
@@ -1012,14 +1012,14 @@ int
                                                     &fbink_cfg,
                                                     NULL);
 
-				// And now, switch to a smaller font size when consuming the script's output...
+				// And now, switch to a smaller font size when consuming the script's output…
 				msg_cfg.padding     = HORI_PADDING;
 				msg_cfg.size_px     = fbink_state.font_h;
 				msg_cfg.margins.top = (short int) rc;
-				// Drop the bottom margin to allow stomping over the status bar...
+				// Drop the bottom margin to allow stomping over the status bar…
 				msg_cfg.margins.bottom = 0;
 
-				LOG(LOG_WARNING, "Listing offending processes...");
+				LOG(LOG_WARNING, "Listing for offending processes…");
```

Comment:
NACK, listing for sounds weird?

Reply:
i misunderstood, then added "all" even though I don't understand how I got that wrong.